docs/syntax/zenuml.md

```diff
@@ -0,0 +1,368 @@
+> **Warning**
+>
+> ## THIS IS AN AUTOGENERATED FILE. DO NOT EDIT.
+>
+> ## Please edit the corresponding file in [/packages/mermaid/src/docs/syntax/zenuml.md](../../packages/mermaid/src/docs/syntax/zenuml.md).
+
+# ZenUML
```

Comment:
Also needs to add zenuml in the sidebar so it's visible.

Reply:
Done.